content/en/developers/dogstatsd/high_throughput.md

```diff
@@ -499,6 +499,18 @@ The number of datagrams dropped because of an error while writing to Datadog.
 : **Metric type**: count<br>
 The number of metrics dropped because the internal receiving channel is full (only when using `WithChannelMode()`). Starting with version `3.6.0` of the Go client.
 
+`datadog.dogstatsd.client.aggregated_context`
+: **Metric type**: count<br>
+The number of contexts aggregated when client side aggregation is enabled. Starting with version `4.3.0`.
+
+`datadog.dogstatsd.client.aggregated_context_by_type`
+: **Metric type**: count<br>
+The number of contexts aggregated by type when client side aggregation is enabled. Starting with version `4.3.0`.
+
+`datadog.dogstatsd.client.metrics_by_type`
+: **Metric type**: count<br>
+The number of metrics by type sent to the DogStatsD client by your application (before sampling). Starting with version `4.3.0`.
+
```

Comment:
Those are behind `WithDevMode` ([here](https://github.com/DataDog/datadog-go/blob/master/statsd/telemetry.go#L141)) option until the next major. Also we should mentionned the tag `metrics_type` so user know how to use it.